tests/integration/components/frost-list-test.js

```diff
@@ -20,18 +23,23 @@ describeComponent(
     }
   },
   function () {
+    beforeEach(function () {
+      initialize()
+    })
+
     it('renders frost-list-item', function () {
       var list = Ember.A()
       list.addObject(Ember.Object.create(server.create('listItem', {
         'dimension': 'custom'
       })))
+      list.addObject(Ember.Object.create(server.create('listItem', {
+        'dimension': 'custom'
```

Comment:
I'd get rid of any reference to `dimension` - we want to move past that concept of needing to provide an explicit type for the record